README.ru.md

```diff
@@ -488,33 +482,33 @@ curl -d '{ "jsonrpc": "2.0", "id": 1, "method": "getrawtransaction", "params": [
 
 Запросите информацию о контракте для нашей вызывающей транзакции, полученной на шаге 4:
```

Comment:
Deployment transaction is not updated above.

Reply:
Fixed.